docs/rules/expect-expect.md

```diff
@@ -1,10 +1,9 @@
 # Enforce assertion to be made in a test body (`expect-expect`)
 
-💼 This rule is enabled in the following
-[configs](https://github.com/jest-community/eslint-plugin-jest/blob/main/README.md#shareable-configurations):
-`all`.
+✅ This rule will _warn_ in the `recommended`
+[config](https://github.com/jest-community/eslint-plugin-jest/blob/main/README.md#shareable-configurations).
```

Comment:
This comes from a new feature to indicate when a rule is set to *warn* in a config.